platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/events/UnsignedEvent.java

```diff
@@ -120,10 +127,11 @@ public static class ClassVersion {
     private static final Set<Long> TRANSACTION_TYPES =
```

Comment:
I think this is no longer needed

Reply:
Its needed in `deserialize`

Reply:
ah, because we did not update the deserialization to not rely on transaction wrappers. this should be done to match the serialization, and then this can be removed

Reply:
changed deserialize